### PR TITLE
chore: move authorize_document .py and .html files to erpnext from bloomstack_core

### PR DESCRIPTION
**Pull request by mrunmay-epi**
**Task ID**: https://bloomstack.com/desk#Form/Task/TASK-2021-00281
**Description**: Added "authorize_document.html" and "authorize_document.py" file ERPNext.
**Dependency PR**: https://github.com/Bloomstack/erpnext/pull/1642
**Bloomstack Core PR**: https://github.com/Bloomstack/bloomstack_core/pull/717

### CODE REVIEW

File: erpnext/www/authorize_document.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
+{% extends "templates/web.html" %}
+
+{% block title %} Authorize Document {% endblock %}
+{% block style %}
+	<link type="text/css" rel="stylesheet" href="/assets/bloomstack_core/css/authorize_document.css">
```
Comment (sahil28297) [inline erpnext/www/authorize_document.html]: path to css is not proper, connect with Vishal for the proper path

**Review by sahil28297 — CHANGES_REQUESTED** [review]

**Comment by sahil28297** [comment]
@mrunmay-epi The dependency PR link is not proper

File: erpnext/www/authorize_document.html (hunk 1 above)
Comment (mrunmay-epi) [reply erpnext/www/authorize_document.html]: Hi Sahil,
The PR has been updated with suggested changes.

PR link: https://github.com/Bloomstack/erpnext/pull/1659

**Review by sahil28297 — APPROVED** [review]

File: erpnext/www/authorize_document.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+from __future__ import unicode_literals
+
+import re
```
Comment (sahil28297) [inline erpnext/www/authorize_document.py]:
```suggestion
```

**Review by sahil28297 — CHANGES_REQUESTED** [review]

**Review by sahil28297 — APPROVED** [review]

**Pull request merged**